### ISSUE TITLE
golem version 

### ISSUE THREAD

**Issue by fb-elong** [RConsortium/submissions-pilot2#40]
The line seems not proper

https://github.com/RConsortium/submissions-pilot2/blob/eee7e9432ce93506d43a37b926cc0efa357764e4/inst/golem-config.yml#L3

**Comment by rpodcast** [comment]
Good catch, I will update this to match the package version in DESCRIPTION.
